integrations/langchain/README.md

```diff
@@ -0,0 +1,109 @@
+## 🦜🔗 Using Unity Catalog AI with Langchain
+
+Integrate Unity Catalog AI package with Langchain to allow seamless usage of UC functions as tools in agents application.
+
+## Installation
+
+```sh
+# install from the source
+pip install git+ssh://git@github.com/serena-ruan/unitycatalog-ai.git#subdirectory=integrations/langchain
+```
+
+> [!NOTE]
+> Once this package is published to PyPI, users can install via `pip install unitycatalog-ai-langchain`
+
+## Get started
+
+### Databricks-managed UC
+
+To use Databricks-managed Unity Catalog with this package, follow the [instructions](https://docs.databricks.com/en/dev-tools/cli/authentication.html#authentication-for-the-databricks-cli) to authenticate to your workspace and ensure that your access token has workspace-level privilege for managing UC functions.
+
+#### Client setup
+Initialize a client for managing UC functions in a Databricks workspace, and set it as the global client.
+
+```python
+from unitycatalog.ai.client import set_uc_function_client
+from unitycatalog.ai.databricks import DatabricksFunctionClient
+
+client = DatabricksFunctionClient(
+    warehouse_id="..." # replace with the warehouse_id
+    cluster_id="..." # optional, only pass when you want to use cluster for function creation
+)
+
+# sets the default uc function client
+set_uc_function_client(client)
```

Comment:
Instead of globally setting the client, can we pass the client to `LangchainToolkit`?

Reply:
We can, but I think the more general use case is that the user uses the same client across multiple toolkits

Reply:
Can the user pass the client to the multiple toolkits?

```
client = ...
t1 = LangchainToolkit(client=client)
t2 = LangchainToolkit(client=client)
t3 = LangchainToolkit(client=client)
```

Reply:
Is there a case that you want to use different clients with different toolkits?

```python
client1 = ...
client2 = ...
t1 = LangchainToolkit(client=client1)
t2 = LangchainToolkit(client=client2)
```

Reply:
Yes we can pass the same client to multiple toolkits; The second case is also possible, pls see https://docs.google.com/document/d/1pNmXvkAt8x70XuvfFJESXlbRN2OA19SWFfW1QU7yokQ/edit?pli=1#bookmark=id.qbel1z2ohit7 for this decision :)